### PR TITLE
[OTA-3968] Basic unit tests setup

### PR DESCRIPTION
**Pull request by piotrsmol**
Signed-off-by: Piotr Smol <piotrsmol.pvt@gmail.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/device/OverviewPanel.jsx [hunk 1]
```diff
@@ -50,15 +48,15 @@ class OverviewPanel extends Component {
...
             <HistoryMtuList device={device} />
-          </TabPane>
-          <TabPane tab={t('devices.installation_pending')} id="queued-packages" key={OVERVIEW_PANEL_TAB_ID_1}>
+          </Tabs.TabPane>
+          <Tabs.TabPane tab={t('devices.installation_pending')} id="queued-packages" key={OVERVIEW_PANEL_TAB_ID_1}>
             <QueueMtuList cancelMtuUpdate={cancelMtuUpdate} />
-          </TabPane>
-          <TabPane tab={t('devices.consent_pending')} id="approval-pending-campaigns" key={OVERVIEW_PANEL_TAB_ID_2}>
+          </Tabs.TabPane>
+          <Tabs.TabPane tab={t('devices.consent_pending')} id="approval-pending-campaigns" key={OVERVIEW_PANEL_TAB_ID_2}>
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/components/device/OverviewPanel.jsx]: Line 57 exceeds the maximum line length of 120  max-len

File: ota-plus-web/app/reactapp/src/pages/Dashboard/__tests__/Dashboard.spec.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Provider } from 'mobx-react';
+import stores from '../../../stores';
+import Dashboard from '..';
+
+it('renders without crashing', () => {
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/pages/Dashboard/__tests__/Dashboard.spec.js]: "should render" will be fine

File: ota-plus-web/app/reactapp/src/components/device/OverviewPanel.jsx (hunk 1 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/components/device/OverviewPanel.jsx]: done

File: ota-plus-web/app/reactapp/src/pages/Dashboard/__tests__/Dashboard.spec.js (hunk 2 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/pages/Dashboard/__tests__/Dashboard.spec.js]: done

**Pull request merged**